components/notifications.js

```diff
@@ -404,16 +499,14 @@ function ItemMention ({ n }) {
       <small className='fw-bold text-info ms-2'>
         your item was mentioned in
       </small>
-      <div>
-        {n.item?.title
-          ? <Item item={n.item} />
-          : (
-            <div className='pb-2'>
-              <RootProvider root={n.item.root}>
-                <Comment item={n.item} noReply includeParent rootText='replying on:' clickToContext />
-              </RootProvider>
-            </div>)}
-      </div>
+      {n.item?.title
+        ? <div className='ps-2'><Item item={n.item} /></div>
+        : (
+          <div className='pb-2'>
+            <RootProvider root={n.item.root}>
+              <Comment item={n.item} noReply includeParent rootText='replying on:' clickToContext />
+            </RootProvider>
+          </div>)}
```

Comment:
The spacing of these notifications caused the bolt to have not padding off the left edge.